CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 --------------------------------------
 #### [1.0.31]
 
+<<<<<<< HEAD
```

Comment:
It seems a merge conflict

Reply:
Yes. It was the remnant of a merge conflict.  Cleaned up now.